pybliometrics/scopus/plumx_metrics.py

```diff
@@ -0,0 +1,150 @@
+        existence of data and applicability of metrics to document type.
+        For details on social media metrics see
+        https://plumanalytics.com/learn/about-metrics/social-media-metrics/
+        """
+        categories = self._json.get('count_categories', [])
+        social_metrics = _category_metrics('socialMedia', categories)
+        return _list_metrics_totals(social_metrics) or None   
+    
+    @property
+    def usage(self):
+        """A list of namedtuples representing Usage category metrics:
+        metrics that capture interaction with and usage of research.
+        Note: Can be empty. Specific metrics can be absent depending on
+        existence of data and applicability of metrics to document type.
+        For details on Usage metrics see
+        https://plumanalytics.com/learn/about-metrics/usage-metrics/
+        """
+        categories = self._json.get('count_categories', [])
+        usage_metrics = _category_metrics('usage', categories)
+        return _list_metrics_totals(usage_metrics) or None     
+    
+    def __init__(self, identifier, refresh=False):
+        """Class to represent the results from retrieval request from the
+        Scopus PlumX Metrics API.
+
+        Parameters
+        ----------
+        identifier : str
+            The identifier of a document. Must be Scopus EID.
+
+        refresh : bool or int (optional, default=False)
+            Whether to refresh the cached file if it exists or not.  If int
+            is passed, cached file will be refreshed if the number of days
+            since last modification exceeds that value.
+
+        Notes
+        -----
+        The directory for cached results is `{path}/ENHANCED/{identifier}`,
+        where `path` is specified in `~/.scopus/config.ini`.
+        """
+        Retrieval.__init__(self,
+                           identifier=identifier,
+                           id_type='elsevierId',
+                           api='PlumXMetrics',
+                           refresh=refresh,
+                           view='ENHANCED')
+
+
+def _category_metrics(category_name, categories_list):
+    """Auxiliary function returning all available metrics in a single
+    category as a list of dicts
+    """
+    cat_counts = []
+    for item in categories_list:
+        if item.get('name') == category_name:
+            cat_counts += item.get('count_types', [])
+    return cat_counts
+
+
+def _list_metrics_totals(metric_counts):
+    """Formats list of dicts of metrics into list of namedtuples in form
+    (name, total)
+    """
+    out = []
+    # Metric names and associated counts if either is not None
+    values = [[m.get('name'), m.get('total')] for m in metric_counts
+              if m.get('name') and m.get('total')]
+    fields = 'name total'
+    capture = namedtuple('Metric', fields)
+    for v in values:
+        new = capture(name=v[0], total=v[1])
+        out.append(new)
+    return out
```

Comment:
Please add a newline at the end and use a class description that corresponds to the new standard ("Interaction with ..."). Otherwise this class looks perfect.